From a6f337fdf68294cfae233724567cbeea0ae5148f Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Fri, 18 Jun 2021 18:07:40 +0300
Subject: [PATCH 188/247] ASoC: atmel-i2s: Set symmetric sample bits

The I2S needs to have the same sample bits for both capture and playback
streams.

Fixes: b543e467d1a9 ("ASoC: atmel-i2s: add driver for the new Atmel I2S controller")
Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Link: https://lore.kernel.org/r/20210618150741.401739-1-codrin.ciubotariu@microchip.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/atmel/atmel-i2s.c | 1 +
 1 file changed, 1 insertion(+)

--- a/sound/soc/atmel/atmel-i2s.c
+++ b/sound/soc/atmel/atmel-i2s.c
@@ -560,6 +560,7 @@ static struct snd_soc_dai_driver atmel_i
 	},
 	.ops = &atmel_i2s_dai_ops,
 	.symmetric_rates = 1,
+	.symmetric_samplebits = 1,
 };
 
 static const struct snd_soc_component_driver atmel_i2s_component = {
